src/components/IPScan/AdvancedOptions/index.tsx

```diff
@@ -142,13 +142,26 @@ export const AdvancedOptions = ({
         </summary>
         <fieldset className={css('new-fieldset')}>
           <legend>Job configuration</legend>
+          <label style={{ marginBottom: '1rem' }}>
+            <ToggleSwitch
+              id="seqtype"
+              name="seqtype"
+              switchCond={false}
+              label="Sequence type:"
+              size="small"
+              offValue="Amino acids"
+              onValue="Nucleotides"
```

Comment:
```suggestion
              offValue="protein (amino acids)"
              onValue="DNA/RNA (nucleotides)"
```

Reply:
does it really work on RNA? is `U` allowed?

Reply:
Yes. `U` is supported.

**Example:**

Input:

```fasta
>Test
ggccgcggccugcugccguuugugcugcuggcgcugggcauugcgccgugggcgguggaa
ggcgcggaaaacgcgcugaaaggcggcgcgugcccgccgcgcaaaauugugcagugccug
cgcuaugaaaaaccgaaaugcaccagcgauuggcagugcccggauaaaaaaaaaugcugc
cgcgauaccugcgcgauuaaaugccugaacccgguggcgauuaccaacccggugaaagug
aaaccgggcaaaugcccggugguguauggccagugcaugaugcugaacccgccgaaccau
ugcaaaaccgauagccagugccugggcgaucugaaaugcugcaaaagcaugugcggcaaa
gugugccugaccccggugaaagcg
```

Output:


```tsv
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	FunFam	G3DSA:4.10.75.10:FF:000001	Anosmin 1	77	126	2.5E-17	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	ProSiteProfiles	PS51390	WAP-type 'four-disulfide core' domain profile.	24	72	13.453233	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	SUPERFAMILY	SSF57256	Elafin-like	25	72	1.57E-10	T	06-08-2024	IPR036645	Elafin-like superfamily	-	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	ProSiteProfiles	PS51390	WAP-type 'four-disulfide core' domain profile.	78	126	14.54461	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Phobius	SIGNAL_PEPTIDE_C_REGION	C-terminal region of a signal peptide.	16	21	-	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	SUPERFAMILY	SSF57256	Elafin-like	74	126	6.54E-13	T	06-08-2024	IPR036645	Elafin-like superfamily	-	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Phobius	SIGNAL_PEPTIDE_H_REGION	Hydrophobic region of a signal peptide.	4	15	-	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	FunFam	G3DSA:4.10.75.10:FF:000001	Anosmin 1	23	72	2.5E-15	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	CDD	cd00199	WAP	76	126	4.92296E-9	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	SignalP_EUK	SignalP-noTM	SignalP-noTM	1	21	-	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	PRINTS	PR00003	4-disulphide core signature	107	116	1.4E-7	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	PRINTS	PR00003	4-disulphide core signature	78	87	1.4E-7	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	PRINTS	PR00003	4-disulphide core signature	100	107	1.4E-7	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	PRINTS	PR00003	4-disulphide core signature	117	125	1.4E-7	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Phobius	NON_CYTOPLASMIC_DOMAIN	Region of a membrane-bound protein predicted to be outside the membrane, in the extracellular region.	22	128	-	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Gene3D	G3DSA:4.10.75.10	-	24	73	9.5E-10	T	06-08-2024	IPR036645	Elafin-like superfamily	-	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	PANTHER	PTHR19441	WHEY ACDIC PROTEIN  WAP	7	125	4.5E-21	T	06-08-2024	IPR050514	WAP four-disulfide core domain	GO:0004867(PANTHER)|GO:0005615(PANTHER)	Reactome:R-HSA-6798695|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	SMART	SM00217	wap2	27	72	1.9E-5	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	SMART	SM00217	wap2	81	126	8.6E-22	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Pfam	PF00095	WAP-type (Whey Acidic Protein) 'four-disulfide core'	81	125	1.1E-7	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Pfam	PF00095	WAP-type (Whey Acidic Protein) 'four-disulfide core'	27	71	4.3E-6	T	06-08-2024	IPR008197	WAP-type 'four-disulfide core' domain	GO:0005576(InterPro)|GO:0030414(InterPro)	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Phobius	SIGNAL_PEPTIDE	Signal peptide region	1	21	-	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Phobius	SIGNAL_PEPTIDE_N_REGION	N-terminal region of a signal peptide.	1	3	-	T	06-08-2024	-	-	-	-
Test_orf6	d1b6cbf29dde9e5220196f3f6114a1c3	128	Gene3D	G3DSA:4.10.75.10	-	74	125	3.6E-15	T	06-08-2024	IPR036645	Elafin-like superfamily	-	Reactome:R-HSA-190373|Reactome:R-HSA-5654726|Reactome:R-HSA-6798695|Reactome:R-HSA-6803157|Reactome:R-HSA-6809371|Reactome:R-HSA-9830364|Reactome:R-MMU-6798695|Reactome:R-MMU-6803157|Reactome:R-RNO-6803157|Reactome:R-SSC-6803157|Reactome:R-SSC-6809371
Test_orf3	289cf7872c2605c6ae224731aa07fb50	42	SignalP_GRAM_POSITIVE	SignalP-TM	SignalP-TM	1	11	-	T	06-08-2024	-	-	-	-
Test_orf9	3b5965af329771e0bcc7c1d6fd4a2496	40	MobiDBLite	mobidb-lite	consensus disorder prediction	15	40	-	T	06-08-2024	-	-	-	-
Test_orf9	3b5965af329771e0bcc7c1d6fd4a2496	40	MobiDBLite	mobidb-lite	consensus disorder prediction	23	40	-	T	06-08-2024	-	-	-	-
Test_orf9	3b5965af329771e0bcc7c1d6fd4a2496	40	SignalP_GRAM_POSITIVE	SignalP-TM	SignalP-TM	1	26	-	T	06-08-2024	-	-	-	-
Test_orf8	3fc751b463e324b7910bbb34f23373b7	86	Phobius	TRANSMEMBRANE	Region of a membrane-bound protein predicted to be embedded in the membrane.	45	63	-	T	06-08-2024	-	-	-	-
Test_orf8	3fc751b463e324b7910bbb34f23373b7	86	Phobius	NON_CYTOPLASMIC_DOMAIN	Region of a membrane-bound protein predicted to be outside the membrane, in the extracellular region.	40	44	-	T	06-08-2024	-	-	-	-
Test_orf8	3fc751b463e324b7910bbb34f23373b7	86	Phobius	TRANSMEMBRANE	Region of a membrane-bound protein predicted to be embedded in the membrane.	21	39	-	T	06-08-2024	-	-	-	-
Test_orf8	3fc751b463e324b7910bbb34f23373b7	86	TMHMM	TMhelix	Region of a membrane-bound protein predicted to be embedded in the membrane.	42	64	-	T	06-08-2024	-	-	-	-
Test_orf8	3fc751b463e324b7910bbb34f23373b7	86	Phobius	CYTOPLASMIC_DOMAIN	Region of a membrane-bound protein predicted to be outside the membrane, in the cytoplasm.	1	20	-	T	06-08-2024	-	-	-	-
Test_orf8	3fc751b463e324b7910bbb34f23373b7	86	Phobius	CYTOPLASMIC_DOMAIN	Region of a membrane-bound protein predicted to be outside the membrane, in the cytoplasm.	64	86	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	Phobius	SIGNAL_PEPTIDE_H_REGION	Hydrophobic region of a signal peptide.	7	17	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	Phobius	SIGNAL_PEPTIDE	Signal peptide region	1	24	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	TMHMM	TMhelix	Region of a membrane-bound protein predicted to be embedded in the membrane.	65	87	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	TMHMM	TMhelix	Region of a membrane-bound protein predicted to be embedded in the membrane.	7	29	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	Phobius	CYTOPLASMIC_DOMAIN	Region of a membrane-bound protein predicted to be outside the membrane, in the cytoplasm.	88	128	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	Phobius	NON_CYTOPLASMIC_DOMAIN	Region of a membrane-bound protein predicted to be outside the membrane, in the extracellular region.	25	64	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	Phobius	SIGNAL_PEPTIDE_N_REGION	N-terminal region of a signal peptide.	1	6	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	Phobius	TRANSMEMBRANE	Region of a membrane-bound protein predicted to be embedded in the membrane.	65	87	-	T	06-08-2024	-	-	-	-
Test_orf11	0aed180405bafcb32e693c5433782622	128	Phobius	SIGNAL_PEPTIDE_C_REGION	C-terminal region of a signal peptide.	18	24	-	T	06-08-2024	-	-	-	-
```